samples/bluetooth/iso_connected_benchmark/src/main.c

```diff
@@ -924,7 +936,7 @@ static int change_central_settings(void)
 
 	printk("Change CIG settings (y/N)? (Current settings: cis_count=%u, "
 	       "interval=%u, latency=%u)\n",
-	       cig_create_param.num_cis, cig_create_param.interval,
+	       cig_create_param.num_cis, cig_create_param.c_to_p_interval,
```

Comment:
While it will currently always be the same, suggest to log both anyways

Reply:
done